crates/python/tests/conftest.py

```diff
@@ -89,3 +91,13 @@ def bell_program() -> str:
 MEASURE 0 ro[0]
 MEASURE 1 ro[1]
 """
+
+
+@pytest.fixture
+def qvm_http_client() -> QVMHTTPClient:
+    return QVMHTTPClient("http://localhost:5000")
+
+
+@pytest.fixture
+def quilc_rpcq_client() -> RPCQClient:
+    return RPCQClient("tcp://localhost:5555")
```

Comment:
For configurability/consistency, might be better to initialize these fixtures with a QCS client, that way they aren't hardcoded.